src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,100 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
 import { formatNumber, Metric } from '../lib';
+import { MetricMeta } from '../lib/load_status';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: any;
+  description: any;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
+      data-test-subj={testSubjectName}
+      title={title}
+      titleSize="xxs"
+      description={description}
+      reverse
     />
   );
 };
 
+/*
+ * Displays a metric with the correct format.
+ */
+export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
+  const { name } = metric;
+  if (name === 'Delay') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={`${name} avg`}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
+              title={formatDelayFooterTitle(metric.meta!)}
+              description={metric.meta!.description}
+            />
+          )
+        }
+      />
+    );
+  } else if (name === 'Load') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={name}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
+              title={metric.meta!.value.join('; ')}
+              description={metric.meta!.description}
+            />
+          )
+        }
+      />
+    );
+  } else if (name === 'Response time avg') {
```

Comment:
I grouped the 	`Response time` metrics together for two reasons:
1. The average makes more sense when one knows what it's relative to (the max). Having the min in here would be nice down the line too.
2. Alls 2 lines of 3 cards rather than 3 lines where the last one only has 2 cards. I guess we could play with the felx on these components but this was easier and more toward Design's recommendations.